account_invoice_check_total/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 Acsone SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Invoice Check Total',
+    'summary': """
+        Check if the verification total is equal to the bill's total""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Acsone SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu/',
```

Comment:
@rven Replace URL as per [this](https://github.com/odoo/odoo/blob/11.0/addons/hr_recruitment/__manifest__.py#L11)

Reply:
@nikul-serpentcs Done